src/components/ItemsList.js

```diff
@@ -101,82 +102,104 @@ const ItemsList = () => {
-                        shoppingItemObject.lastPurchasedOn,
-                      )}
-                    />
-                    <label
-                      htmlFor={shoppingItemObject.shoppingListItemName}
-                      aria-label={getItemDescription(
-                        shoppingItemObject.daysLeftForNextPurchase,
-                      )}
+    <div className="max-h-screen flex flex-col box-border items-center">
+      <header className="bg-green-400 w-full fixed text-center">
+        <h2 className="pt-6 pb-16 text-4xl font-thin text-gray-100">
+          Your Shopping List
+        </h2>
+        <span className="text-white top-0 right-0 absolute sm:mt-4 sm:mr-4">
+          <HomeIcon />
+        </span>
+      </header>
+      <main className="bg-white relative w-full h-full mt-24 rounded-t-3xl overflow-auto">
+        {loading && (
+          <img className="m-auto w-12" src={spinner} alt="Loading..." />
+        )}
+        {error && <p>An error has occured...</p>}
+        {listHasNoItems && (
+          <div className="h-64 bg-white flex flex-col w-screen justify-center items-center text-gray-900">
+            <p className="">You haven't created a shopping list yet...</p>
+            <button
+              className="bg-white-100 px-6 py-3 text-sm mt-6 border border-color-gray-500 border-solid rounded shadow-md hover:bg-green-500 cursor-pointer hover:text-white"
+              type="submit"
+              onClick={handleRedirect}
+            >
+              Add First Item
+            </button>
+          </div>
+        )}
+        {listHasAtLeastOneItem && (
+          <div className="mt-6 max-w-md mx-auto overflow-auto">
+            <SearchBar
+              value={searchTerm}
+              setValue={(searchTerm) => {
+                setSearchTerm(searchTerm);
+              }}
+            />
+            <ul className="mt-4 mb-2 mx-2">
+              {shoppingList[0].items
+                .filter((shoppingItemObject) =>
+                  shoppingItemObject.shoppingListItemName
+                    .toLowerCase()
+                    .includes(searchTerm.toLowerCase()),
+                )
+                .sort(sortShoppingList)
+                .map((shoppingItemObject, index) => {
+                  const shopIndex = shoppingList[0].items.indexOf(
+                    shoppingItemObject,
+                  );
+                  return (
+                    <li
+                      className="py-3 mt-2 rounded-lg flex items-center shadow-md"
+                      key={shoppingItemObject.shoppingListItemName + index}
+                      style={{
+                        backgroundColor: getShoppingItemBackgroundStyles(
+                          shoppingItemObject.daysLeftForNextPurchase,
+                          getDaysBetweenCurrentAndPreviousPurchase,
+                          shoppingItemObject.lastPurchasedOn,
+                        ),
```

Comment:
I believe we are missing a line-through when the item is purchased.

Maybe you can do a ternary to indicate if `wasItemPurchasedWithinLastOneDay` then have a line-through, otherwise don't have it. You can take it further and instead of line-through maybe create a special style for items that were purchased last 24 hours (`wasItemPurchasedWithinLastOneDay`). Something similar to the disabled style.

Reply:
I will definitly try that out. Yes, I just could not figure how to write the `[input:checkbox]checked + label` in Tailwind ;) That's why this effect is still missing.

Reply:
You can do something like: 

`className={` ${checked ? 'line-through' : ' '}`}`

in terms of `line-through` you can either create a pure **CSS** style and call it `line-through` or come up with some special styling for the items that were purchased within the last 24 hours and replace `line-through` name with some other styling name. Or you can directly write tailwind class names inside. 

Let me know if that makes sense 🙂

Reply:
Made the changes with pure CSS for now, but still considering finding another way for the next issue.